db/migrate/20230308042434_backfill_auto_bump_cooldown_category_setting.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class BackfillAutoBumpCooldownCategorySetting < ActiveRecord::Migration[7.0]
+  def up
+    execute(<<~SQL)
```

Comment:
Tested this locally. I had 33 categories and 5 category settings:

```
SELECT COUNT(*) FROM categories;
#=> 33

SELECT COUNT(*) FROM category_settings;
#=> 5
```

Afterwards I have 33 category_settings:

```
SELECT COUNT(*) FROM category_settings;
#=> 33
```